src/test/scala/is/hail/methods/LinearMixedRegressionSuite.scala

```diff
@@ -14,7 +14,8 @@ import org.testng.annotations.Test
 class LinearMixedRegressionSuite extends SparkSuite {
 
   def assertDouble(a: Annotation, value: Double, tol: Double = 1e-6) {
-    assert(D_==(a.asInstanceOf[Double], value, tol))
+    val d1 = a.asInstanceOf[Double]
+    assert(D_==(d1, value, tol), s"$d1 did not equal $value with tol $tol")
```

Comment:
This method is duplicated in all the regression suites, let's take this chance to get rid of the duplication. Put it in utils.